<?xml version="1.0"?>
<launch>
    <include file="$(find assignment_2_2022)/launch/sim_w1.launch" />
    <param name="des_pos_x" value= "0.0" />
    <param name="des_pos_y" value= "1.0" />
    
    <!--Parameter to set the frequency the info is printed with-->
    <param name="frequency" type="double" value="1.0" />
    
    <node pkg="assignment_2_2022" type="wall_follow_service.py" name="wall_follower" />
    <node pkg="assignment_2_2022" type="go_to_point_service.py" name="go_to_point"  />
    <node pkg="assignment_2_2022" type="bug_as.py" name="bug_action_service" output="screen" />
    <node pkg="assignment_2_2022" type="control.py" name="control" output="screen" launch-prefix="xterm -hold -e" />
    <node pkg="assignment_2_2022" type="counter.py" name="counter"  />
    <node pkg="assignment_2_2022" type="rob_info.py" name="rob_info" output="screen" launch-prefix="xterm -hold -e" />
</launch>
